engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -23,7 +23,7 @@ import scala.util.{Try, Failure, Success}
 object ValidateActor {
   private val tag = "ValidateActor"
 
-  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson], workflowOptions: Option[WdlJson]): Props = {
+  def props(wdlSource: WdlSource, wdlJson: Option[WdlJson] = None, workflowOptions: Option[WdlJson] = None): Props = {
```

Comment:
@Horneth You're going from one horrible extreme to another (not horrible) extreme here. The (well, A)  problem with WA is that it owns too many responsibilities, not that it doesn't have everything it needs as a parameter. However having a single responsibility doesn't mean it's literally doing *one* thing, just that it does one type of thing which can be parameterized